formatter/src/formatter/node.rs

```diff
@@ -51,7 +51,34 @@ impl Formatter<'_> {
     }
 
     pub fn node_name(&mut self, name: &NodeName) {
-        self.printer.word(name.to_string());
+        match name {
+            NodeName::Path(_) | NodeName::Punctuated(_) => self.printer.word(name.to_string()),
+            NodeName::Block(block) => {
+                // Special case for props spreading with an empty element name
+                if let [syn::Stmt::Expr(syn::Expr::Range(range), _)] = block.stmts.as_slice() {
+                    match &range.end {
+                        // Spreading with identifier `{..ident}`
+                        Some(end) => {
+                            if let syn::Expr::Path(path) = &**end {
+                                if let Some(segment) = path.path.segments.last() {
+                                    let formatted = format!("{{..{}}}", segment.ident);
+                                    self.printer.word(formatted);
+                                    return;
+                                }
+                            }
+                        }
+                        // Spreading without identifier `{..}`
+                        None => {
+                            self.printer.word("{..}");
+                            return;
+                        }
+                    }
+                }
+
+                // Default handling if it doesn't match any specific pattern
+                self.printer.word(name.to_string())
+            }
+        }
```

Comment:
There is already a formatter method that formats node value blocks ;)

```suggestion
        if let NodeName::Block(block) = name {
            self.node_value_block_expr(block, false, false);
        } else {
            self.printer.word(name.to_string());
        }
```

Reply:
Good, I was sure I missed something better.